src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -183,6 +183,8 @@ if(CLR_CMAKE_TARGET_WIN32)
         list(APPEND PREPROCESS_DEFINITIONS -DTARGET_AMD64)
     elseif (CLR_CMAKE_HOST_ARCH_ARM64)
         list(APPEND PREPROCESS_DEFINITIONS -DTARGET_ARM64)
+    elseif (CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
I think whole changes in this file can be moved to https://github.com/dotnet/runtime/pull/62886 to make this PR sef-contained.